### ISSUE TITLE
sup loss and unsup loss is not the same order of magnitude

### ISSUE THREAD

**Issue by rankeey** [google-research/uda#36]
INFO:tensorflow:step: 10(global step: 10)	sample/sec: 24.330	loss: 0.606	top-1: 0.750	unsup_loss: 0.010
INFO:tensorflow:step: 20(global step: 20)	sample/sec: 23.123	loss: 0.757	top-1: 0.500	unsup_loss: 0.021
INFO:tensorflow:step: 30(global step: 30)	sample/sec: 23.740	loss: 0.830	top-1: 0.250	unsup_loss: 0.019
INFO:tensorflow:step: 40(global step: 40)	sample/sec: 24.192	loss: 0.708	top-1: 0.500	unsup_loss: 0.016
INFO:tensorflow:step: 50(global step: 50)	sample/sec: 24.254	loss: 0.987	top-1: 0.250	unsup_loss: 0.017
INFO:tensorflow:step: 60(global step: 60)	sample/sec: 23.358	loss: 0.733	top-1: 0.750	unsup_loss: 0.018

the loss is sup_loss + unsup_loss，it seems the sup_loss and unsup_loss is not the same order of magnitude， is it reasonable?
Or it means I should change uda_coeff from 1 to 10? 
Or any other suggestion?

**Comment by qizhex** [comment]
The unsup_loss will first increase then decrease. At first, the model may make similar predictions for different examples so the consistency loss is low. After several epochs, the model's prediction will become more diverse due to the supervised loss so the consistency loss will grow.

**Comment by rankeey** [comment]
Thanks for your reply.

**Comment by spiritbreake** [comment]
Related to text classification, unsup loss is increasing which I think should decrease after some update steps. I think unsup loss should decrease after some training steps because the model has learnt a lot and should output similar probability distribution for a sample and it's augmented example.
It will be helpful if anyone could explain why unsup_loss should increase or decrease. 

![Screenshot 2019-09-30 at 12 48 18 PM](https://user-images.githubusercontent.com/50858510/65857036-a67c6500-e380-11e9-8cd6-514b7ee9f47e.png)

**Comment by guotong1988** [comment]
Thank you.
